src/harness/fourslashImpl.ts

```diff
@@ -1341,7 +1341,7 @@ namespace FourSlash {
             }));
         }
 
-        public verifyQuickInfoAt(markerName: string | Range, expectedText: string, expectedDocumentation?: string, expectedTags?: {name: string; text: string;}[]) {
+        public verifyQuickInfoAt(markerName: string | Range, expectedText: string, expectedDocumentation?: string, expectedTags?: { name: string; text: string; }[]) {

```

Comment:
please undo these formatting changes. We'll create a giant formatting PR when we (someday, maybe) start using a formatter.

Reply:
Okay. Keep in mind that this is just the formatting the langserver did automatically, and not a special formatter.

Reply:
In the final PR I can remove it, I was just tired of constantly removing it when it was done automatically, when my main concern was getting tests to pass.